<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QWidget" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineForSources"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForBufer"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForDevices"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForRequests"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForLambda"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForAlpha"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineForBeta"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Количество источников</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Размер буфера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Количество приборов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Количество заявок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Лямбда</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Альфа</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Бета</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>211</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="optimalButton">
      <property name="text">
       <string>Задать оптимальные параметры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backToMenuButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
